blob/src/main/java/io/crate/blob/BlobContainer.java

```diff
@@ -91,21 +87,22 @@ private void createSubDirectories(File parentDir) {
         }
     }
 
-    public interface FileVisitor {
-
-        boolean visit(File file) throws IOException;
-
+    public Iterable<File> getFiles() {
+        final Collection<File> files = new LinkedList<>();
```

Comment:
@mfussenegger do you think it's worth to implement a custom Iterable here even when `directory.listFiles()` returns an array of files?